test/e2e/deploy/volumeleaks.go

```diff
@@ -48,8 +48,16 @@ func GetHostVolumes(d *Deployment) map[string][]string {
 		// Intentional Output instead of CombinedOutput to dismiss warnings from stderr.
 		// lvs may emit lvmetad-related WARNING msg which can't be silenced using -q option.
 		out, err := ssh.Output()
-		if err != nil && os.IsNotExist(err) {
-			break
+		if err != nil {
+			if os.IsNotExist(err) {
+				// ssh wrapper does not exist: all node handled.
+				break
+			}
+			ginkgo.By(fmt.Sprintf("ssh on node %d returned error '%s', retry once", worker, err.Error()))
+			out, err = ssh.Output()
+			if err != nil {
+				ginkgo.By(fmt.Sprintf("ssh on node %d repeatedly returned error '%s', give up on that node", worker, err.Error()))
+			}
```

Comment:
This message is not going to be visible easily when the assertion fails later on. Wouldn't it be better to fail the test here?

Reply:
yes, better idea is to fail there as job will fail anyway